daemon/containerd/image_exporter.go

```diff
@@ -113,6 +113,8 @@ func (i *ImageService) ExportImage(ctx context.Context, names []string, outStrea
 				"target": target,
 			}).Debug("export image without name")
 		}
+
+		i.LogImageEvent(target.Digest.String(), target.Digest.String(), events.ActionSave)
```

Comment:
SO confusing that it's "image expert", but actually "image save" 😞

Reply:
I had to check 3 times to make sure it's the right one...

Reply:
The image is not exported at this point yet. We should log the events after `Export` returns a nil error.

Reply:
This is the way graph drivers work too. Also you could export multiple images…

so it’s more “ready to export this image” rather than “this image was exported”

Reply:
Oh, right. That's weird though.. but if that's how it worked with the graphdrivers then it should stay that way. Sorry!

Reply:
It is weird. I’ll give you that. I even did what you said at one point but reverted. It would be too big of a change. Maybe we can fix this in a followup, fix both implementations if we think it’s useful

Reply:
Yes, we should map what events we trigger "where" and with what information. Many of them where just added "willy-nilly" in the code, but not all were properly defined.

Should be slightly easier to find all locations now that we have the consts defined, but it could use some tweaking (and documenting) overall